src/components/MyAccount/SettingsTab/PasswordChangeForm.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react"
+import {
+  Form,
+  FormField,
+  TextInput,
+  Button,
+} from "@nypl/design-system-react-components"
+import styles from "../../../../styles/components/MyAccount.module.scss"
+import { BASE_URL } from "../../../config/constants"
+import type { Patron } from "../../../types/myAccountTypes"
+
+const PasswordChangeForm = ({
+  patron,
+  updateModal,
+}: {
+  patron: Patron
+  updateModal: (errorMessage?: string) => void
+}) => {
+  const [formData, setFormData] = useState({
+    oldPassword: "",
+    newPassword: "",
+    confirmPassword: "",
+    passwordsMatch: true,
+  })
+
+  const validateForm = () => {
+    return (
+      formData.oldPassword !== "" &&
+      formData.newPassword !== "" &&
+      formData.confirmPassword !== "" &&
+      formData.passwordsMatch
+    )
+  }
+
+  const handleInputChange = (e) => {
```

Comment:
A more universal way of getting HTML input values is by using the `name` attribute. What you're doing here checking against the `id` works but use `name` instead. This also means you should add `name="oldPassword"` to the `TextInput` components (but update the values accordingly).

Reply:
@7emansell just need this included for the updates.